Dockerfile.jupyter_kernel_tests

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
I would move this file under `/etc/build/DockerFile.jupyter_kernel_tests` or rather than having to build the image... run the `jupyter/pyspark-notebook` directly and do all the steps in the CMD of the `docker run ... bash -c 'all steps here``.